modules/nf-core/fgbio/zipperbams/main.nf

```diff
@@ -0,0 +1,57 @@
+process FGBIO_ZIPPERBAMS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::fgbio=2.0.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fgbio:2.0.2--hdfd78af_0' :
+        'quay.io/biocontainers/fgbio:2.0.2--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(ubam)
+    tuple val(meta), path(mbam)
+    path(fasta)
+    path(dict)
+
+    output:
+    tuple val(meta), path("*_zipped.bam"), emit: bam
```

Comment:
if you set `_zipped` as a suffix in this you also need to hard code this in your command, otherwise it won't be suffixed by `_zipped` all the time

Reply:
thanks! I've actually fixed the problem in a completely dynamic way.